pkg/fakerp/server.go

```diff
@@ -170,7 +190,7 @@ func (s *Server) handlePut(w http.ResponseWriter, req *http.Request) {
 	s.write(oc)
 
 	// simulate Context with property bag
-	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Hour)
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
```

Comment:
aligned with create context

Reply:
Please revert back to 1h, this is also responsible for upgrades.